tests/lib.rs

```diff
@@ -36,10 +36,71 @@ fn smoke() {
 }
 
 #[test]
+fn remove_range() {
+    let ranges = [0, 1, 63, 64, 65, 100, 4096 - 1, 4096, 4096 + 1, 65536 - 1, 65536, 65536 + 1];
+    for (i, &a) in ranges.iter().enumerate() {
+        for &b in &ranges[i..] {
+            let mut bitmap = RoaringBitmap::from_iter(0..65536 + 1);
+            assert_eq!(bitmap.remove_range(a..b), (b - a) as u64);
```

Comment:
Could you add an assertion here that the bitmap contains the correct values, something like

```rust
assert_eq!(bitmap, RoaringBitmap::from_iter((0..a).chain(b..65536 + 1)));
```

to make sure it's the correct values being removed and there's no off by one or something in the removal.